packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -73,18 +64,12 @@ function MyFeedHeading({
           }
           loading={isRefreshing}
         >
-          {!isMobile && !isNewMobileLayout ? 'Refresh feed' : null}
+          {isLaptop ? 'Refresh feed' : null}
```

Comment:
Just wondering but should this not match the variant above?
(so !isMobile)

Reply:
The expression would be `!isMobile && isLaptop` which would only be true if `isLaptop` is true, thus changed to `isLaptop`. If we would say only `!isMobile` that would include tablet as well, which the above expression is not including.